src/OrchardCore/OrchardCore.Users.Core/Services/UserService.cs

```diff
@@ -370,4 +374,40 @@ public async Task<IUser> RegisterAsync(RegisterUserForm model, Action<string, st
 
         return user;
     }
+
+    /// <inheritdoc/>
+    public async Task<bool> EnableAsync(IUser user)
+    {
+        ArgumentNullException.ThrowIfNull(user, nameof(user));
+
+        (user as User).IsEnabled = true;
+
+        var result = await _userManager.UpdateAsync(user);
+
+        if (result.Succeeded)
+        {
+            var userContext = new UserContext(user);
+            await _handlers.InvokeAsync((handler, context) => handler.EnabledAsync(context), userContext, _logger);
+        }
+
+        return result.Succeeded;
+    }
+
+    /// <inheritdoc/>
+    public async Task<bool> DisableAsync(IUser user)
+    {
+        ArgumentNullException.ThrowIfNull(user, nameof(user));
```

Comment:
```suggestion
        ArgumentNullException.ThrowIfNull(user);
```

Reply:
What's wrong with this?

Reply:
without the second paramters, it'll have the same result. We don't use the second parameter anywhere else